src/NServiceBus.Core/Sagas/SagaIdGeneratorContext.cs

```diff
@@ -5,7 +5,7 @@ namespace NServiceBus.Sagas
     /// <summary>
     /// Context provided to the saga id generator.
     /// </summary>
-    public class SagaIdGeneratorContext : ContextBag
+    public class SagaIdGeneratorContext : IExtendable
```

Comment:
Does anyone remeber the original intent of that interface? I like the fact that it makes the naming consistent (to avoid the ugly context.Context we have on a few other contexts)